torch_xla/experimental/spmd_fully_sharded_data_parallel.py

```diff
@@ -0,0 +1,107 @@
+from typing import (Any, Callable, Optional, Union)
+
+import torch
+import torch.nn as nn
+from torch._prims_common import TensorLike, TensorSequenceType
+
+import numpy as np
+
+import torch_xla
+import torch_xla.distributed.spmd as spmd
+
+def _prepare_spmd_partition_spec(param):
+  partition_spec = [None] * len(param.shape)
+  # Skip scalar tensors.
```

Comment:
"Skip scalar tensor and call it replicated."

Reply:
Fixed.